fedimint-server/src/consensus/mod.rs

```diff
@@ -131,6 +138,11 @@ pub async fn run(
         );
     }
 
+    let checkpoint_retention: u64 = env::var(FM_DB_CHECKPOINT_RETENTION_ENV)
+        .unwrap_or(FM_DB_CHECKPOINT_RETENTION_DEFAULT.to_string())
+        .parse()
+        .expect("FM_DB_CHECKPOINT_SESSION_DIFFERENCE var is invalid");
```

Comment:
Previous name still here? Possibly a sign to use the const here too.

Reply:
Fixed, added constant as well